process_request/helpers.py

```diff
@@ -3,6 +3,102 @@
 from rapidfuzz import fuzz
 
 
+def get_collection_creator(resource):
+    """Takes json for an archival object that has the _resolved parameter on resource::linked_agents. Iterates through linked_agents; if the role is creator, appends to list, and returns list as a string."""
+    creators = []
+    if resource.get("linked_agents"):
+        for linked_agent in resource.get("linked_agents"):
+            if linked_agent.get("role") == "creator":
+                creators.append(linked_agent.get("_resolved").get('display_name').get('sort_name'))
+    return ",".join(creators)
+
+
+def get_location(top_container_info):
+    """Gets a human-readable location string for a top container
+
+    Args:
+        top_container_info (dict): json for a top container (with resolved container locations)
+
+     Returns:
+         string: all locations associated with the top container, separated by a comma.
+     """
+    locations = []
+    for c in top_container_info.get("container_locations"):
+        locations.append(c.get("_resolved").get("title"))
+    return ",".join(locations)
+
+
+def get_container_field(top_container_info, container_field):
+    """Gets the value from a top container field
+
+    Args:
+        top_container_info (dict): json for a top container
+        container_field (str): top container field
+
+    Returns:
+        string: value of the top container field
+    """
+    return(top_container_info.get(container_field))
+
+
+def check_for_instance_type(archival_object, type_to_check):
+    """Gets the index value of a specific instance type for an archival object
+
+    Args:
+        archival_object (dict): json for an archival object
+        type_to_check (str): instance type to check against
+
+    Returns:
+        int: index of the matching instance type in the list of instances
+    """
+    list_of_instances = []
+    for i in archival_object.get("instances"):
+        instance_type = i.get("instance_type")
+        list_of_instances.append(instance_type)
+    if type_to_check in list_of_instances:
+        return list_of_instances.index(type_to_check)
```

Comment:
Can we remove this code from this PR? It's a little hard to evaluate what changes are actually being made otherwise.